app/data/tasks/deduplicate_records.py

```diff
@@ -0,0 +1,185 @@
+from django.conf import settings
+from django.core.paginator import Paginator
+
+from celery import shared_task
+from celery.utils.log import get_task_logger
+
+from data.models import (DedupeJob, RecordDuplicate)
+from ashlar.models import Record
+
+import datetime
+from pytz import timezone
+
+logger = get_task_logger(__name__)
+
+
+def get_time_allowance():
+    return datetime.timedelta(hours=settings.DEDUPE_TIME_RANGE_HOURS)
+
+
+# .001 degrees is ~110m give or take 10 depending on location, which should
+# be good enough
+def get_distance_allowance():
+    return settings.DEDUPE_DISTANCE_DEGREES
+
+
+def calculate_score(record, duplicate):
+    """
+    Set a score representing the likelyhood the two records are duplicated, based
+    on the criteria.  score is a float between 0 and 1
+
+    normalized dimensions are considered equal (1 day equally as important as 100m)
+    """
+    tdelta = abs(record.occurred_from - duplicate.occurred_from)
+    tscore = 0
+    if tdelta.total_seconds() == 0:
+        tscore = 1.0
+    else:
+        tscore = tdelta.total_seconds() / get_time_allowance().total_seconds()
```

Comment:
Is this logic backwards? It looks like the score increases the farther apart the times are, when it seems it should decrease, since 1.0 is a perfect match.